libraries/Microsoft.Bot.Builder.AI.QnA/QnAMakerOptions.cs

```diff
@@ -92,5 +92,14 @@ public QnAMakerOptions()
         /// <seealso cref="RankerTypes"/>
         [JsonProperty("rankerType")]
         public string RankerType { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether  enable Precise Answer.
```

Comment:
Grammar. Copyedit.

Reply:
Fixed.

---
In reply to: [430553266](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r430553266) [](ancestors = 430553266)